brainstormforce/astra
# refactor: Added 2px border support for outline GB buttons in all cases

### Description
- As per requirement 2px border will apply to all outline GB buttons

### Types of changes
- Dynamic CSS changes for outline buttons adjusting border size

### How has this been tested?
- Test this with Customizer > Global > Buttons - Border width
- Check with empty border size (in this case border should be 2px for outline GB buttons)
- Check with '0' border size (in this case border should be 2px for outline GB buttons)
- Check with 'x' number border size (in this case border should be 'x' px for outline GB buttons)

### Checklist:
